app/models/service_ansible_playbook.rb

```diff
@@ -53,17 +53,10 @@ def job(action)
     service_resources.find_by(:name => action, :resource_type => 'OrchestrationStack').try(:resource)
   end
 
-  def postprocess(action)
-    inventory_raw_id = options.fetch_path(job_option_key(action), :inventory)
-    delete_inventory(action, inventory_raw_id) if inventory_raw_id
-    log_stdout(action)
```

Comment:
My understanding is that you remove all code of creating/removing inventory because no more tower is used. Don't we still want to log output on error?

Reply:
Sure, I can move this line into `#on_error`.

Reply:
Done.